CMakeLists.txt

```
@@ -2050,6 +2051,11 @@ elseif(APPLE)
     set(HAVE_SDL_FILE TRUE)
   endif()
 
+  if(IOS OR TVOS OR MACOSX OR DARWIN)
+    file(GLOB SDL_VIDEO_CAPTURE_SOURCES ${SDL3_SOURCE_DIR}/src/video/SDL_video_capture_apple.m)
+    list(APPEND SOURCE_FILES ${SDL_VIDEO_CAPTURE_SOURCES})
```

The `SOURCES` variable does not exist anymore.
```suggestion
    sdl_sources("${SDL3_SOURCE_DIR}/src/video/SDL_video_capture_apple.m")
```